src/components/button/button.spec.js

```diff
@@ -50,6 +50,20 @@ describe("Button", () => {
     );
   });
 
+  describe("when iconType specified with no children", () => {
+    it("icon matches the style for an icon only button", () => {
+      const wrapper = mount(<Button iconType="bin" iconPosition="center" />);
```

Comment:
You can remove the `iconPosition="center"` here

Reply:
✅